DS-Assignment6/Employee.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Assignment 1
+ * Class for storing the information regarding the employee
+ * @author Arpit Pittie
+ */
+package session6;
+
+public class Employee implements Comparable<Employee>{
+
+	private String empId;		//ID of the employee
+	private String empName;		//Name of the employee
+	private String empAddress;	//Address of the employee
+	
+	public Employee(String id, String name, String address) {
+		this.empId = id;

```

Comment:
There seems to be some issue with the alignment. It taking too much space from left of these inner code lines.